makeinfo-6.3 allows @itemx only after @item and fails the build as:
    gcc-2.95.3/gcc/install.texi:1607: @itemx must follow @item

--- a/gcc/install.texi
+++ b/gcc/install.texi
@@ -1604,7 +1604,7 @@ linker that are fixed by using the GNU versions of these tools.
 Embedded PowerPC system in little endian mode for use in running under
 the PSIM simulator.
 
-@itemx powerpcle-*-eabi
+@item powerpcle-*-eabi
 Embedded PowerPC system in little endian mode.
 
 You can specify a default version for the @samp{-mcpu=}@var{cpu_type}
